plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/cleaner/DockerAbandonedResourcesCleaner.java

```diff
@@ -96,23 +96,27 @@ public void run() {
      * Cleans up CHE docker containers which don't tracked by API any more.
      */
     @VisibleForTesting
-    void cleanContainers() {
+    void cleanContainers(@Nullable @Named("che.server_id") String cheServerId) {
```

Comment:
Looks like this code doesn't compile since this method is called without arguments